content/developer/howtos/localization.rst

```diff
@@ -0,0 +1,370 @@
+    :start-at: chart_at_template_0010
+    :end-at: </record>
+
+
+Account groups
+--------------
+
+.. seealso::
+   :ref:`Account Group References <reference/account_group>`
+
+Account groups allows to describe structure of chart of accounts.
+
+.. example::
+  `addons/l10n_il/data/account.group.template.csv <https://github.com/odoo/odoo/blob/15.0/addons/l10n_il/data/account.group.template.csv>`__.
+
+  .. csv-table::
+     :file: {ODOO_ABSPATH}/addons/l10n_il/data/account.group.template.csv
+     :widths: 20,20,20,20,20
+     :header-rows: 1
+
+Taxes
+-----
+
+.. seealso::
+   - :ref:`Tax References <reference/account_tax>`
+   - :doc:`/applications/finance/accounting/taxation/taxes/taxes`
+
+To add taxes you first need to specify tax groups. It has just two fields: name and country. Create file `data/account_tax_group_data.xml` and list the groups:
+
+.. code-block:: xml
+
+    <?xml version="1.0" encoding="utf-8"?>
+    <odoo>
+        <data noupdate="1">
+            <record id="tax_group_tva_0" model="account.tax.group">
+                <field name="name">TVA 0%</field>
+                <field name="country_id" ref="base.ch"/>
+            </record>
+
+            ...
+        </data>
+    </odoo>
+
+.. example::
+  `addons/l10n_ch/data/account_tax_group_data.xml <https://github.com/odoo/odoo/blob/15.0/addons/l10n_ch/data/account_tax_group_data.xml>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_ch/data/account_tax_group_data.xml
+    :language: xml
+    :start-after: <data
+    :end-before: </data>
+
+.. example::
+  `addons/l10n_uk/data/account.tax.group.csv <https://github.com/odoo/odoo/blob/15.0/addons/l10n_uk/data/account.tax.group.csv>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_uk/data/account.tax.group.csv
+    :language: csv
+
+
+Now you can add the taxes via `data/account_tax_template_data.xml` file.
+
+
+.. example::
+  `addons/l10n_ae/data/account_tax_template_data.xml <https://github.com/odoo/odoo/blob/15.0/addons/l10n_ae/data/account_tax_template_data.xml>`__.
+
+  .. literalinclude:: {ODOO_RELPATH}/addons/l10n_ae/data/account_tax_template_data.xml
+    :language: xml
+    :start-at: uae_sale_tax_5_dubai
+    :end-at: </record>
+
+If some accounts should use default taxes, you can set them up in `data/account_account_template_post_data.xml`
+
+Tax Report
+----------
+
+.. seealso::
+   - :ref:`Tax Report Line References <reference/account_tax_report_line>`
+   - :doc:`/applications/finance/accounting/reporting/declarations/tax_returns`
+
+In previous section you noticed fields `invoice_repartition_line_ids` / `refund_repartition_line_ids` and probably understand nothing about it. Good news: you are not alone on this incomprehension. Bad news: you have to figure it out a bit. The topic is complicated indeed:
+
+.. graphviz:: images/tax_report.dot
+
+
+But lucky we have a presentation explaining the tax reports in details:
+
+.. youtube:: PuXE_NyFRTM
+    :align: right
+    :width: 700
+    :height: 394
```

Comment:
As I said on the other PR, I'm not sure it's a good idea to link it here. This was done for v13, and the model has changed a little since then. Moreover, it essentially compares the financial reports with the new engine, and I'm not sure it'll help newcomers in any way.

Reply:
It is indeed explaining the difference (we don't care about that part), but also explaining how the new model works (we care about this)

Reply:
But on the other hand, it's already partly outdated, as account.tax.report objects didn't exist back then. Running exactly the examples from the presentation won't work. I'm not 100% against keeping it, but we need to keep that in mind. And I'm not sure we can say it explicitly here either, like "oh, we know we're giving you something a bit outdated, but who cares, we didn't want to make a new one" :p Maybe there is a more diplomatic way, though.